### PR TITLE
Add policy rules to verify RPM sig keys

### PR DESCRIPTION
**Pull request by lcarva**
Ref: EC-842

### CODE REVIEW

**Comment by lcarva** [comment]
In draft because https://github.com/release-engineering/rhtap-ec-policy/pull/36 must be merged first.

**Comment by lcarva** [comment]
Dependencies merged. Moved out of draft.

File: policy/release/rpm_signature.rego [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	some key in _allowed_rpm_signature_keys
+	not _is_valid_key(key)
+	msg := sprintf("Unexpected format of signing key %q", [key])
+}
+
+_is_valid_key(key) if {
+	# "unsigned" is a special value that indicates no signatures.
+	key == "unsigned"
+} else if {
+	# Rego's regex processor doesn't seem to handle "pattern". So we do it "manually" here :(
```
Comment (zregvart) [inline policy/release/rpm_signature.rego:124]: This is a bit unclear to me, did you mean?
```suggestion
	# Rego's JSON Schema processor doesn't seem to handle "pattern". So we do it "manually" here :(
```

**Review by zregvart — APPROVED** [review]

File: policy/release/rpm_signature.rego (hunk 1 above)
Comment (lcarva) [reply policy/release/rpm_signature.rego]: Yup. Thank you

**Pull request merged**